src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -57,7 +57,7 @@ public void execute_validIndexFilteredList_success() {
 
         String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_EATERY_SUCCESS, eateryToDelete);
 
-        Model expectedModel = new ModelManager(model.getAddressBook(), model.getFeedList(), new UserPrefs());
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), model.getFeedList(), new UserPrefs());
```

Comment:
why you use modelmanager instead of model ah isn't model the interface class

Reply:
idk the previous test case, like the one above uses modelmanager for expected model, so i just changed it ahhh

Reply:
ah okie